<launch>  
  <!--include file="/home/bubu/simulation_ws/src/my_simulations/launchscales_world.launch" /-->
  <node name="gazebo_transport_to_ros_topic_frps" pkg="scale_frps_pkg" type="gazebo_transport_to_ros_topic_frps"
        output="screen" launch-prefix="xterm -e" >
  <param name="gazebo_transport_topic_to_sub_frps" type="str" value="/gazebo/default/weight_scales/joint_02/force_torque_frps/wrench" />
  <param name="ros_topic_to_pub_frps" type="str" value="gazebo_transport_to_ros_topic_frps" />
  <param name="ros_rate" type="double" value="5.0" />
  </node>
</launch>
